src/common/mem.c

```diff
@@ -173,28 +209,39 @@ umempobj_open(const char *path, const char *layout_name, int flags, struct umem_
 			}
 		}
 
-		if (store != NULL)
-			umm_pool->up_store = *store;
 		umm_pool->up_priv = pop;
 		return umm_pool;
-	}
+	case DAOS_MD_BMEM:
+		/* TODO mmap tmpfs file */
+		/* TODO Load all meta pages from SSD */
+		/* TODO Replay WAL */
```

Comment:
My original idea is that umempobj_open() calls umem common function to do mmap first, then load meta blob and replay WAL, the last step is to call dav_obj_open() or ad_blob_open() according to the daos_md_backend.

Reply:
make sense to me, now BMEM do the open+mmap internally, we can refine it at following patch.